charts/rucio-daemons/templates/renew-fts-cronjob.yaml

```diff
@@ -5,14 +5,14 @@
   {{- if or (eq .Values.ftsRenewal.vo "atlas") (eq .Values.ftsRenewal.vo "dteam") }}
   - name: longproxy
     secret:
-      secretName: {{ .Release.Name }}-longproxy
+      secretName: {{ if empty .Values.ftsRenewal.longProxy.existingSecret.name }} {{ .Release.Name }}-longproxy {{- else }} {{ .Values.ftsRenewal.longProxy.existingSecret.name | quote }} {{ end }}
   {{- else }}
   - name: usercert
     secret:
-      secretName: {{ .Release.Name }}-fts-cert
+      secretName: {{ .Values.ftsRenewal.ftsCert.existingSecret.name | quote }}
```

Comment:
Actually, this also is a breaking change. You should do the same as for the longproxy